data_analysis.ipynb

```diff
@@ -4,19 +4,1336 @@
    "cell_type": "markdown",
```

Comment:
<p>nit: it would be better to have legend for the lines in this plot (I assume they represent different years)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepakkrish212/MATH327-Final/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yes, but it looked ugly and covered the graph fully. That's why I removed it